modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/requests/RequestBodyTest.java

```diff
@@ -237,6 +237,7 @@ public Response methodWithRequestBodyWithoutAnnotation(
         @Path("/user")
         @Operation(summary = "Delete user",
                 description = "This can only be done by the logged in user.")
+        @Consumes({"application/json", "application/xml"})
```

Comment:
why has this be added? this is not needed for the test and should be removed I'd say

Reply:
Done!